vhdbuilder/packer/init-variables.sh

```diff
@@ -240,33 +240,50 @@ fi
 
 # windows image sku and windows image version are recorded in code instead of pipeline variables
 # because a pr gives a better chance to take a review of the version changes.
+WINDOWS_IMAGE_PUBLISHER="MicrosoftWindowsServer"
+WINDOWS_IMAGE_OFFER="WindowsServer"
 WINDOWS_IMAGE_SKU=""
 WINDOWS_IMAGE_VERSION=""
+WINDOWS_IMAGE_URL=""
+IMPORTED_IMAGE_NAME=""
 # shellcheck disable=SC2236
 if [ ! -z "${WINDOWS_SKU}" ]; then
 	source $CDIR/windows-image.env
 	case "${WINDOWS_SKU}" in
 	"2019"|"2019-containerd")
 		WINDOWS_IMAGE_SKU=$WINDOWS_2019_BASE_IMAGE_SKU
 		WINDOWS_IMAGE_VERSION=$WINDOWS_2019_BASE_IMAGE_VERSION
+		IMPORTED_IMAGE_NAME="windows-2019-imported-${CREATE_TIME}-${RANDOM}"
```

Comment:
What about Gen 2? Is it like windows-2019-gen2-imported-.....?

Reply:
It is a temporary name so it does not care gen1 or gen2